lnbits/core/views/api.py

```diff
@@ -657,3 +659,23 @@ async def _generator(stream: BytesIO):
             "Expires": "0",
         },
     )
+
+
+@core_app.get("/api/v1/audit/")
+async def api_auditor(wallet: WalletTypeInfo = Depends(get_key_type)):
+    if wallet.wallet.user not in LNBITS_ADMIN_USERS:
+        raise HTTPException(
+            status_code=HTTPStatus.FORBIDDEN, detail="Not an admin user"
+        )
+
+    total_balance = await get_total_balance()
+    error_message, node_balance = await WALLET.status()
```

Comment:
- should check the value of `error_message`
- if the node cannot be accessed then (maybe) an exception should be raised

Reply:
Now if node is down it just gives zero in node_balance_msats. Hesitating if it's better to see an exception or at least lnbits_balance_msats. I'm open for opinions.

Reply:
I think this is fine as is and edge cases are out of scope for this PR. Eventually this can be used to build a security kill switch for LNbits which I'm very much looking forward to.